coveo-testing/tests_testing/mock_module/inner.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any, Callable
+
+
+def inner_function() -> str:
+    return "Un pangolin ça marche comme M.Burns."
```

Comment:
Oh wow! It does!

![](https://media.giphy.com/media/H4zzl9nKke0s4vNaBm/giphy-downsized.gif)